backend/data/processors/manual_fixes/CVENFixes.py

```diff
@@ -0,0 +1,163 @@
+"""
+https://github.com/csesoc/Circles/wiki/Manual-Fixes-to-Course-Prerequisites
+
+Copy this into a new file for the relevant faculty's fixes:
+e.g. COMPFixes.py, ACCTFixes.py, PSYCFixes.py
+
+Apply manual [code] fixes to processed conditions in conditionsProcessed.json so
+that they can be fed into algorithms.
+
+If you make a mistake and need to regenerate conditionsProcessed.json, then you
+can run:
+    python3 -m data.processors.conditionsPreprocessing
+
+To then run this file:
+    python3 -m data.processors.manualFixes.[CODE]Fixes
+"""
+
+from data.utility import data_helpers
+
+# Reads conditionsProcessed dictionary into 'CONDITIONS'
+CONDITIONS = data_helpers.read_data("data/final_data/conditionsProcessed.json")
+PROCESSED = "processed"
+
+# Reads coursesProcessed dictionary into 'COURSES' (for updating exclusions)
+COURSES = data_helpers.read_data("data/final_data/coursesProcessed.json")
+
+def fix_conditions():
+    """ Functions to apply manual fixes """
+
+    # TODO: call your functions here
+    for course in ("CVEN2303", "CVEN3303", "CVEN3304"):
+        CONDITIONS[course] = CVEN_2303_3303_3304()
+
+    for course in ("CVEN4002", "CVEN4003"):
+        CONDITIONS[course][PROCESSED] = CVEN_4002_4003()
+
+    CONDITIONS["CVEN4050"][PROCESSED] = CVEN_4050()
+    CONDITIONS["CVEN4106"][PROCESSED] = CVEN_4106()
+
+    for course in ("CVEN4201", "CVEN4202", "CVEN4204"):
+        CONDITIONS[course][PROCESSED] = CVEN_4201_4202_4204()
+
+    CONDITIONS["CVEN4300"][PROCESSED] = CVEN_4300()
+    CONDITIONS["CVEN4308"][PROCESSED] = CVEN_4308()
+    CONDITIONS["CVEN4309"][PROCESSED] = CVEN_4309()
+    CONDITIONS["CVEN4507"][PROCESSED] = CVEN_4507()
+    CONDITIONS["CVEN4951"] = CVEN_4951()
+    CONDITIONS["CVEN9826"] = CVEN_9826()
+
+    # Updates the files with the modified dictionaries
+    data_helpers.write_data(
+        CONDITIONS, "data/final_data/conditionsProcessed.json")
+    data_helpers.write_data(COURSES, "data/final_data/coursesProcessed.json")
+
+# TODO: implement your functions here
+def CVEN_2303_3303_3304():
+    """
+    "original": "Prerequisites: (ENGG1300 AND ENGG2400) OR (CVEN1300 AND CVEN2301)<br/>(Pre-requisite only applicable to UG cohort taking this course).<br/><br/>",
+    "processed": "(ENGG1300 && ENGG2400) || (CVEN1300 && CVEN2301) (only applicable to UG cohort taking this course)"
+    """
+
+    return {
+        "original": "Prerequisites: (ENGG1300 AND ENGG2400) OR (CVEN1300 AND CVEN2301)<br/>(Pre-requisite only applicable to UG cohort taking this course).<br/><br/>",
+        "processed": "(ENGG1300 && ENGG2400) || (CVEN1300 && CVEN2301)",
+        "warning": "Requirements only applicable to undergrads"
+    }
+
+def CVEN_4002_4003():
+    """
+    "original": "Pre Req: 132UOC<br/><br/>",
+    "processed": "Req: 132UOC"
```

Comment:
hmm i dont like this being a manual fix. I would prefer that you try to match it on conditions_preprocessing.py:
https://github.com/csesoc/Circles/blob/dev/backend/data/processors/conditions_preprocessing.py
this will save you a whole lot of lines of code here.

Reply:
This should be fixed with some changes I've made to the regex that recognises "prerequisite: "